### PR TITLE
Enable Floorplan and Rules by default.

### LINKED ISSUE #382: Enable Rules Engine & Floorplan by default
As a user I want the rules engine and floorplan experiments to be enabled by default so I don't have to enable them manually

Leave speech UI disabled by default for now while we develop it further

### PR DESCRIPTION
**Pull request by mrstegeman**
Fixes #382

### CODE REVIEW

**Comment by hobinjk** [comment]
I think the intent is to have them enabled by default but still disable-able
@benfrancis ?

**Comment by mrstegeman** [comment]
So they're still intended to be experiments for now, then?

**Comment by benfrancis** [comment]
> I think the intent is to have them enabled by default but still disable-able

Correct, sorry if that wasn't clear.

**Comment by mrstegeman** [comment]
This version does the trick.

File: src/db.js [hunk 1]
```diff
@@ -113,15 +113,54 @@ var Database = {
...
+        if (baseKey !== '') {
+          newKey = baseKey + '.' + key;
         } else {
-          console.log('Saved default user ' + defaultUser.email);
+          newKey = key;
         }
-      });
+        
+        if (typeof obj[key] === 'object') {
+          Array.prototype.push.apply(settings,
```
Comment (hobinjk) [inline src/db.js]: `settings = settings.concat(generateSettings(obj[key], newKey))` or `settings.push(...generateSettings(obj[key], newKey))` would be a bit cleaner than push.apply

Comment (mrstegeman) [reply src/db.js]: I specifically avoided settings.concat to avoid copying the array unnecessarily. I can switch to spread syntax, though.

**Comment by hobinjk** [comment]
Thanks! LGTM

**Pull request merged**